oidc_apis/scopes.py

```diff
@@ -97,6 +103,45 @@ def scope_address(self):
         return super().scope_address()
 
 
+class SuomiFiUserAttributeScopeClaimsMeta(type):
+    def __dir__(cls):
+        names = super().__dir__()
+        for level in SuomiFiAccessLevel.objects.all():
+            names.append('info_suomifi_' + level.shorthand)
+        return names
+
+    def __getattr__(cls, name):
+        match = re.match(r'^info_suomifi_(.*)', name)
+        if match:
+            try:
+                level = SuomiFiAccessLevel.objects.get(shorthand=match.group(1))
+                return (_(level.name), _(level.description))
```

Comment:
Using `ugettext` or `ugettext_lazy` (the `_` here) doesn't work with dynamic content, since the argument to ugettext must be a static string literal so that it can be picked up for translation. You should rather use some statically translated name in the model like an enumeration or alternatively provide translated fields in the model, e.g. with Django Parler (check e.g. `services/models.py` for an example).

Reply:
Added translations to model.